examples/monitor/metricAlert.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitor.azure.jet.crossplane.io/v1alpha1
+kind: MetricAlert
+metadata:
+  name: example-ma
+spec:
+  forProvider:
+    resourceGroupNameRef:
+      name: example
+    scopes:
+      - "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/example/providers/Microsoft.DBforPostgreSQL/servers/example-psqlserver"
+    criteria:
+      - metricNamespace: "Microsoft.DBforPostgreSQL/servers"
```

Comment:
For learning purposes (no actions needed on our side apart from maybe understanding), what happens if the metric criteria's namespace does not match scoped resource's provider?

Reply:
When the metric criteria's namespace does not match scoped resource's provider, the following error has been observed:

> message: 'create failed: cannot apply: apply failed: creating or updating metric
>     alert "example-ma" (resource group "sergen"): insights.MetricAlertsClient#CreateOrUpdate:
>     Failure responding to request: StatusCode=400 -- Original Error: autorest/azure:
>     Service returned an error. Status=400 Code="ResourceNotFound" Message="{\"code\":\"BadRequest\",\"message\":\"Detect
>     invalid value: Microsoft.Sql/servers for query parameter: ''metricnamespace'',
>     the value must be: Microsoft.DBforPostgreSQL/servers if the query parameter
>     is provided, you can also skip this optional query parameter.\"} Activity
>     ID: 17d8e2db-1968-40b4-b83a-650e2efab7a7.": : File name: main.tf.json'